shared-ui/components/meet-the-team/tooltip/ToolTip.styles.ts

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { StyledToolTipProps } from '../../../lib/types';
+import { fonts } from '../../../style/typography';
+
+const StyledToolTipText = styled.div`
+display: inline-block;
+text-align: left;
+line-height: 2em;
+font-family: ${fonts.nunitoSansSemibold};
+`;
+
+const StyledToolTip = styled.div<StyledToolTipProps>`
+  width: 12em;
+  background-color: ${(props): string => (props.team === 'Directors' ? '#F2C94C': 
+  props.team ==='Tech' ? '#2F80ED' : props.team==='Design' ? '#BB6BD9' : 
+  props.team === 'Social Outreach' ? '#F2A32B' : '#27AE60')};
```

Comment:
For all of the colors, can you do ${colors.WHATEVERCOLOR} instead of putting the string? Since we already have all the colors stored in the colors const